redbadger/continous-compliance
# Compress compliance folder

Closes #19 

## How to test it

- Go to the following [`count-dracula` action](https://github.com/redbadger/count-dracula/runs/2304853947?check_suite_focus=true)
- Press the button **Re-run jobs**
- Make a cuppa 🍵 and wait until *continous compliance* step is done
- Open the accordion and check if the messages read like this

```txt
Creating Compliance folder on compliance.
Compliance folder created 📂.
Copying test results from web/test-results into compliance folder compliance
Test results are copied into compliance folder 🧪 
Compressing compliance folder and naming it 1617962397.zip 🗜
```
![](https://media.giphy.com/media/Oxj0LbZabRWeI/giphy.gif)
